scripts/SANS/sans/command_interface/ISISCommandInterface.py

```diff
@@ -1076,6 +1069,8 @@ def PlotResult(workspace, canvas=None):
         @param canvas: optional handle to an existing graph to write the plot to
         @return: a handle to the graph that was written to
     """
+    raise NotImplementedError("Plotting is not implemented for workbench yet, please contact the dev team.")
```

Comment:
Same here regarding the forum

Reply:
The worst part is I thought, "I'm sure I've written this more than once", then promptly didn't check :face_with_head_bandage: